tasks/carto.py

```diff
@@ -490,6 +500,10 @@ class OBSMeta(Task):
         AND numer_ctag.column_id = numer_c.id
         AND numer_ctag.tag_id = numer_tag.id
         AND numer_c.id = leftjoined_denoms.all_numer_id
+        AND NOT EXISTS (
```

Comment:
We may want to add a check to remove the non-denominated versions of the universe numerators:

`AND NOT (numer_agg IN ('median', 'average') AND denom_id IS NULL)` or something along those lines.

**Edit**: There are no NULL denom_id entries in the `obs_meta` table (except if a column has no denominators at all). This idea doesn't work.